product_uom_category_workflow_policy/views/product_uom_categ_view.xml

```diff
@@ -0,0 +1,84 @@
+                        <button name="action_valid" type="object" string="Valid" class="oe_highlight" attrs="{'invisible':['|',('state','!=','confirm'),('valid_ok','=',False)]}" confirm="Validate data. Are you sure?"/>
+                        <button name="action_restart" type="object" string="Set to Draft" attrs="{'invisible':['|',('state','=','draft'),('restart_ok','=',False)]}" confirm="Restart data. Are you sure?"/>
+                        <field name="state" string="State" widget="statusbar" statusbar_visible="draft,confirm,valid"/>
+                    </header>
+                </xpath>
+
+                <xpath expr="//group" position="after">
+                    <notebook>
+                        <page name="policy" string="Policy" groups="base.group_system">
+                            <group name="policy_1" colspan="4" col="2">
+                                <field name="confirm_ok"/>
+                                <field name="valid_ok"/>
+                                <field name="restart_ok"/>
+                            </group>
+                        </page>
+                        <page name="log" string="Log">
+                            <group name="group_log_1" colspan="4" col="3">
+                                <field name="confirm_date" string="Confirm"/>
+                                <field name="confirm_user_id" nolabel="1"/>
+                                <field name="valid_date" string="Validate"/>
+                                <field name="valid_user_id" nolabel="1"/>
+                                <field name="restart_date" string="Restart"/>
+                                <field name="restart_user_id" nolabel="1"/>
+                            </group>
+                        </page>
+                    </notebook>
+                </xpath>
+            </data>
+            </field>
+        </record>
+
+        <record id="product_uom_categ_view_tree" model="ir.ui.view">
+            <field name="name">product.uom.categ tree</field>
+            <field name="model">product.uom.categ</field>
+            <field name="arch" type="xml">
+                <tree>
+                    <field name="name"/>
+                    <field name="state"/>
+                    <field name="active"/>
+                </tree>
+            </field>
+        </record>
+
+        <record id="product_uom_categ_view_search" model="ir.ui.view">
+            <field name="name">product.uom.categ search</field>
+            <field name="model">product.uom.categ</field>
+            <field name="arch" type="xml">
+                <search>
+                    <field name="name"/>
+                    <filter string="Active" domain="[('active','=',True)]"/>
+                    <filter string="In Active" domain="[('active','=',False)]"/>
```

Comment:
Tambahkan filter ALL:
`<filter string="All" domain="[('active','in',[True, False])]"/>`

Reply:
done